public/expressions/constants.ts

```diff
@@ -9,3 +9,5 @@ export const ORIGIN_PLUGIN_VIS_LAYER = 'anomalyDetectionDashboards';
 export const VIS_LAYER_PLUGIN_TYPE = 'Anomaly Detectors';
 
 export const TYPE_OF_EXPR_VIS_LAYERS = 'vis_layers';
+
+export const OVERLAY_ANOMALIES = 'overlay_anomalies"';
```

Comment:
nit: can you consume this constant in `overlay_anomalies.ts` file as well. There is a `name` value there. We can set it to this constant instead of redefining there.

Reply:
updated `overlay_anomalies.ts` file with this constant